src/mds/Server.cc

```diff
@@ -3327,6 +3327,8 @@ CInode* Server::prepare_new_inode(MDRequestRef& mdr, CDir *dir, inodeno_t useino
     auto _xattrs = CInode::allocate_xattr_map();
     decode_noshare(*_xattrs, p);
     dout(10) << "prepare_new_inode setting xattrs " << *_xattrs << dendl;
+    if (_xattrs->count("encryption.ctx"))
```

Comment:
extremely minor nit: enclose within `{}`

Reply:
Sure, will fix. I think we probably also want to use a string_view literal here.

Reply:
MDS uses `mempool::mds_co::string`

Reply:
Meh, I think it's not worth it. I just left it as a bare string.